test/integration/Session.test.js

```diff
@@ -62,7 +68,20 @@ describe('Session', () => {
             .then((sessionToken) => {
                 assert(sessionToken)
             }))
-        it('should fail to get token with no authentication', (done) => clientNone.session.getSessionToken()
+        it('should get token if set with a token', () => clientSessionToken.session.getSessionToken()
```

Comment:
Would these tests be better off as unit tests? They're not making any requests to other systems.